x-pack/plugins/alerts/server/routes/create.ts

```diff
@@ -38,6 +38,7 @@ export const bodySchema = schema.object({
     }),
     { defaultValue: [] }
   ),
+  notifyOnStateChange: schema.boolean({ defaultValue: false }),
```

Comment:
since the schema is set up this way, I wonder if we could just remove some/most of the cases where we are passing this in explicitly as `false`.

Reply:
I'm actually also wondering if the default should be `true`.  IIRC, and I might well be remembering wrong, but I thought we actually wanted this to be `true`, in that we were just going to make it behave this way, with no user option! :-).

Here's the use case I'm thinking about, which includes the "resolved" action group.  Let's say you have an alert going off, fairly often, so it's noisy, so you throttle it back so you only get 1/10 as many notifications.  Wouldn't you still want a notification when it resolves?  Seems like you wouldn't get one in this case.